java/query/query-runner/src/main/java/sleeper/query/runner/output/WebSocketResultsOutput.java

```diff
@@ -41,10 +45,14 @@ public class WebSocketResultsOutput extends WebSocketOutput implements ResultsOu
 
     private final List<ResultsOutputLocation> outputLocations = new ArrayList<>();
     private final Long maxBatchSize;
+    private final Gson serde;
 
-    public WebSocketResultsOutput(Map<String, String> config) {
+    public WebSocketResultsOutput(Schema schema, Map<String, String> config) {
         super(config);
 
+        this.serde = new GsonBuilder()
+                .registerTypeAdapter(Record.class, new RecordJSONSerDe.RecordGsonSerialiser(schema))
+                .create();
```

Comment:
We should probably add a test that would fail without this. I can see why it would be difficult, since we don't have a way to run integration tests with a real web socket, especially with AWS API Gateway in the middle.

One option would be to make the FakeWebSocketClient wrap around this class and retrieve the messages this class publishes. That way we could test the web socket client and the web socket output together without needing a real web socket.

That should probably be a separate issue. It would take a bit of refactoring.